ui/pages/confirm-transaction/confirm-transaction.component.js

```diff
@@ -78,7 +80,7 @@ export default class ConfirmTransaction extends Component {
     }
   }
 
-  componentDidUpdate(prevProps) {
+  async componentDidUpdate(prevProps) {
```

Comment:
[REQ] I'm sure React warns about making these methods async. Instead convert the callsite to use `.then()`

Reply:
Changes submitted!